app/views/whois_record/inactive.erb

```diff
@@ -5,10 +5,14 @@
 Estonia .ee Top Level Domain WHOIS server
 
 Domain:
-name:       <%= json['name'] %>
+name:                  <%= json['name'] %>
 <%- for st in Array(json['status']) -%>
-status:     <%= st %>
+status:                <%= st %>
```

Comment:
I guess they shouldn't be here?

Reply:
That's intentional to make equal indentation before key and value if keys are short like 'name' and long like 'registration deadline'.

Reply:
![image](https://user-images.githubusercontent.com/7688888/81386886-31031f00-912f-11ea-9d00-68a75ff9a78b.png)